index.js

```diff
@@ -77,22 +81,56 @@ var tss;
             return path.dirname(require.resolve('typescript'));
         };
         TypeScriptSimple.prototype.getDefaultLibFilename = function (options) {
-            if (options.target === 2 /* ES6 */) {
+            if (options.target === ts.ScriptTarget.ES6) {
```

Comment:
> What your compiler option keeps enum expressions?

Its a bug in the language service. Constant enums should *always* be inlined. 

I use atom-typescript and that build using the language service (instead of the cli). : https://atom.io/packages/atom-typescript